src/object/obj_class.c

```diff
@@ -235,6 +235,20 @@ static struct daos_obj_class daos_obj_classes[] = {
 			},
 		},
 	},
+	{
+		.oc_name	= "ec_k2p1_len32k",
+		.oc_id		= DAOS_OC_EC_K2P1_L32K,
```

Comment:
TBH, this ID already already include the information that  data cell = 2, Parity = 1, cell size = 32k. Why do we still store these information below. Though this is not this patch issue.

Reply:
The same redundancy is present in the object classes representing replicated objects.